engine/access/rest/common/parser/transaction.go

```diff
@@ -110,11 +110,14 @@ func (t *Transaction) Parse(raw io.Reader, chain flow.Chain) error {
 		Authorizers:        auths,
 		PayloadSignatures:  payloadSigs.Flow(),
 		EnvelopeSignatures: envelopeSigs.Flow(),
+	})
+	if err != nil {
+		return fmt.Errorf("invalid payer address: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("could not construct transaction body: %w", err)
```
This is a general error, right -- not necessarily the payer address